components/script/dom/htmlmediaelement.rs

```diff
@@ -793,6 +793,9 @@ impl PauseIfNotInDocumentTask {
 }
 
 impl Runnable for PauseIfNotInDocumentTask {
+
```

Comment:
Can you remove the extra line here. We have a lint for empty lines following a `{` so this will not pass the test.